cmd/osbuild-worker/config.go

```diff
@@ -52,6 +52,10 @@ type authenticationConfig struct {
 
 type containersConfig struct {
 	AuthFilePath string `toml:"auth_file_path"`
+	Domain       string `toml:"domain"`
+	Account      string `toml:"account"`
```

Comment:
Maybe we calls this `Path` to have the same nomenclature as the containers library? Maybe also a `Default` prefix to indicate that this is indeed only used if not otherwise specified in the `Source` argument.

Reply:
Can you please clarify what you think should be changed. Just the `Account` to `Path`, or consolidate the two into `Path`?
I took the name `Domain` from here: https://github.com/containers/image/blob/main/docker/reference/normalize.go#L93. This function also uses `officialRepoName`, but I though `Account` is better that `RepoName` (since I am never sure how to call this part of the url).
As for the `Default` prefix, sure, I'll make that change once we clarify the above